### ISSUE TITLE
Editing list bullet in multi-cursor mode is counter-intuitive

### ISSUE THREAD

**Issue by LambdaAlpha** [yzhang-gh/vscode-markdown#829]
## Problem

as title

## How to reproduce

make a list

```
- a
- b
```

put multi-cursor just before a and b, (`|` for cursor)

```
- |a
- |b
```

press backspace.

expected:

```
  |a
  |b
```
actually:

```
  |a
- |b
```

**Comment by yzhang-gh** [comment]
Thanks. However, I'm not sure whether we should support multi-cursor for list editing features.

**Comment by Lemmingh** [comment]
Perhaps, when in multi-cursor state, falling back to [VS Code's default behavior](https://code.visualstudio.com/docs/editor/codebasics#_multiple-selections-multicursor) would be more intuitive.

```markdown
<!-- Original: -->
- |a
- |b

<!-- Backspace: -->
-|a
-|b

<!-- Backspace again: -->
|a
|b
```

```markdown
<!-- Original: -->
- |a
- |b

<!-- Tab: (tabSize = 2) -->
-   |a
-   |b

<!-- Tab again: -->
-     |a
-     |b
```

**Comment by yzhang-gh** [comment]
Sounds like a good idea 👍

**Comment by yzhang-gh** [comment]
I have disabled the list editing feature for multiple cursors in the [dev build](https://github.com/yzhang-gh/vscode-markdown#latest-development-build). Let's leave this as a feature request.

**Comment by yzhang-gh** [comment]
released in v3.4.2
